cmd/main.go

```diff
@@ -142,7 +165,18 @@ func main() {
 		zap.Duration("stream_success_period_auth", envConfig.StreamSuccessPeriod.Auth),
 	)
 
-	// Start the gops agent
+	// List pods from default namespace
+	pods, err := clientset.CoreV1().Pods("illumio-cloud").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		klog.Error(err, "Failed to list pods")
+		return
+	}
+
+	for _, pod := range pods.Items {
+		klog.Info("Pod found ", "name ", pod.Name)
```

Comment:
1. Info level is too verbose. How is this information useful? What is the point of all this?
1. Why use `klog` instead of `logger`? Was that for testing? If so, remove it before merging.